docs/download_files.md

```diff
@@ -91,7 +91,39 @@ Every download also includes the individual [QC report](#qc-report) and, if appl
 
 ## Custom datasets
 
-<!--TODO: Fill in this section with information about creating and downloading custom datasets-->
+To download data for individual samples, multiple projects, or any combination of individual samples and projects, use the `Add to Dataset` button. 
+This process creates a custom dataset for download, referred to as `My Dataset` within the Portal.
+The `My Dataset` button can be used to view and download the custom dataset as a single zip file. 
+
+<!--TODO: Confirm that these are the correct names that will be used as the folder names for data format-->
+Each zip file will be named with a unique dataset ID, the chosen data format (either `SingleCellExperiment` or `AnnData`), and the date you accessed the data on the ScPCA Portal.
```

Comment:
`SingleCellExperiment` and `AnnData` will be displayed as `SINGLE-CELL-EXPERIMENT` and `ANN-DATA` in the folder names. Not sure if you wanted to refer only to the data formats or would also like to present them as they would appear in the folder name.

Reply:
> `SingleCellExperiment` and `AnnData` will be displayed as `SINGLE-CELL-EXPERIMENT` and `ANN-DATA` in the folder names. Not sure if you wanted to refer only to the data formats or would also like to present them as they would appear in the folder name.

I thought we were making them lowercase?

Reply:
Also I don't think we want to use `ANN-DATA` and it should instead be `ANNDATA` without the hyphen.